service/src/main/java/gov/nasa/pds/api/registry/model/api_responses/Pds4ProductBusinessObject.java

```diff
@@ -156,7 +96,7 @@ public void setResponse(List<Map<String, Object>> hits, Summary summary, List<St
     // Products
     for (Map<String, Object> hit : hits) {
       // TODO complete that
-      id = (String) hit.get("_id");
+      id = (String) hit.get("lidvid");
```

Comment:
Is this correct?  I recall @jordanpadams suggesting that `lidvid` was not a universally-reliable key

Reply:
Yes you are right, I went the short way. I will double check that we don t have the '_id' in the dictionary.

Reply:
You are right and I double checked in the code if we can get the id otherwise but unfortunately using the lidvid as id is pretty much baked in the registry-api everywhere.
The reason for that is that we use the "source" part of the opensearch document when we extract the products from opensearch, and doing so we loose the opensearch id.
I will add a "icebox" ticket to redesign that.

Reply:
It is a bit more than that. The lidvid is used as the opensearch `id`. Since `_id` is always returned (lidvid can be filtered out of source through user input as you noted) it is the only guaranteed way of obtaining the lidvid. Therefore you cannot really unbake the `_id` need because you need it and you have no other guaranteed way of obtaining it.